fuse_optimizers/test/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# CORE GTESTS ======================================================================================
+set(TEST_TARGETS
+  # test_fixed_lag_ignition
+  # test_optimizer
+  test_variable_stamp_index
+)
+
+foreach(test_name ${TEST_TARGETS})
```

Comment:
nit, looks like the loop and `TEST_TARGETS` variable are no longer necessary. I'd recommend removing them